CMake/BuildFlatBuffers.cmake

```diff
@@ -64,6 +64,18 @@ function(build_flatbuffers flatbuffers_schemas
     set(FLATC flatc)
   endif()
 
+  # Passed in as env vars
+  if(FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS)
+    set(FLATC_SCHEMA_ARGS
+      ${FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS}
+      --gen-mutable
+      )
+  else()
+    set(FLATC_SCHEMA_ARGS
```

Comment:
Would be slightly cleaner if you defined `FLATC_SCHEMA_ARGS` outside (before) the if, and then made the set inside the if just append to the existing variable.